cmd/krew/cmd/info.go

```diff
@@ -74,6 +74,9 @@ func printPluginInfo(out io.Writer, plugin index.Plugin) {
 	if plugin.Spec.Version != "" {
 		fmt.Fprintf(out, "VERSION: %s\n", plugin.Spec.Version)
 	}
+	if plugin.Spec.Homepage != "" {
```

Comment:
let's reorder this as

- name
- version
- homepage
- description
- caveats

what do you think?

Reply:
OK. I'll change the order as you said.

Reply:
Not related to this PR, but the shortDescription field seems that does not be contained in the output fields in spite of required field.